build.gradle

```diff
@@ -30,8 +30,6 @@ repositories {
   }
```

Comment:
Can we remove the `jcenter()` repository now?  I think that is where we were loading the com.simprints:LibSimprints dependency.

Reply:
Not yet, we need it for "com.noveogroup.android:check" dependency that makes static check. 
Have you used another library for the same checks?

Reply:
LOL, went looking for why the latest version of android:check was not in mavenCentral and found [this issue](https://github.com/noveogroup/android-check/issues/43) logged by someone familiar....

Seems like we need to keep the jcenter() repo for now, but since android:check is unmaintained now, I have logged a new issue to migrate off of that plugin: https://github.com/medic/cht-android/issues/242

Reply:
Thank you! :)